dcpy/models/geospatial/parquet.py

```diff
@@ -0,0 +1,65 @@
+from pyarrow.parquet import FileMetaData
```

Comment:
@sf-dcp - I cleaned this up and dropped the complicated projjson, and I think this shows a good example of how I'd like to use this. See [this line](https://github.com/NYCPlanning/data-engineering/pull/857/files#diff-6de4d995b5b185d7999972afa1df26cb154ba629c4c6c5defaa7216a25eda219R137) in tests. It's nice to load a dict into an object where then I can quickly use computed properties that would be tougher to parse using dicts, and just quickly get the string "EPSG:4326" from the geoparquet metadata.